app/assets/javascripts/miq_flash.js

```diff
@@ -35,6 +39,17 @@ function add_flash(msg, level, options) {
   var alertDiv = $('<div class="' + cls.alert + '"><button class="close" data-dismiss="alert"><span class="pficon pficon-close"></span></button></div>');
   alertDiv.append(iconSpan, textStrong);
 
+  // if options.long alert is given than add a `See More` button to the alert div to allow user the get more details of the error/alert.
+  if (options.long_alert) {
+    var detailsButtonTxt = _("See More");
+    var detailsButton = $('<div style="padding-top: 4px"><button id="alert_expand_button" class="btn btn-default pull-right" type="submit">' + detailsButtonTxt + '</button></div>');
```

Comment:
We can't use `button` or `input type=submit`, because most often, a flash message is *inside* a form. So, instead of expanding, you'd expand *and* submit the form.

(Or, we need to explicitly `event.stopPropagation()`)

Reply:
@himdel just the be clear - we cannot use button **at all**? or is it just the `type=submit` that's causing the issue here?

Reply:
Aah.. ok... `button` with `type=submit` is just a `button`, `type` is not actually a valid attribute on a button element, only on an input.

So... it's not that we can't use either, it's just that we'd have to stop the event from propagating any further.

But... most likely the best bet would be to use `<a href="" class="btn btn-default">` since that one will never submit a form.

EDIT: or btn-link if this is supposed to be a link now :)

Reply:
@himdel nice! :+1: